client/app/core/tagging.service.js

```diff
@@ -1,5 +1,5 @@
 /** @ngInject */
-export function taggingService(CollectionsApi, lodash, exception) {
+export function TaggingService(CollectionsApi, lodash, exception) {
```

Comment:
doh! my :we yes: fail me, http://eslint.org/docs/rules/no-unused-vars can be configured for no-unused arguments